### DISCUSSION TITLE
Release v0.9.1 / 5.51.1 (pre-release)

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#1068] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This build is a Test build, testing the new functionality to use Windows Filtering Platform (WFP) to implement a per sandbox firewall.
This functionality needs to be enabled in the global Sandboxie settings, and the driver needs to be reloaded (or the PC rebooted) for the feature to be activated. Once this is done the firewall rules which can be configured in the network options of each sandbox, will be enforced by the driver.

See release 0.9.0 for more details about this feature: https://github.com/sandboxie-plus/Sandboxie/releases/tag/0.9.0a

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

# Changelog

### Added
- added tray icon indicating no connection to the driver if it happens
- added option to customize the tray icon
- added "DllSkipHook=some.dll" option to disable installation of hooks into selected DLLs
- added localization support for Plus installer (by yfdyh000 and mpheath) [#923](https://github.com/sandboxie-plus/Sandboxie/pull/923)

### Changed
- reworked NtClose handling for better performance and extendibility
- improved tray box menu and list

### Fixed
- fixed issue with fake admin and some NSIS installers [#1052](https://github.com/sandboxie-plus/Sandboxie/issues/1052)
- fixed more issued with FileDispositionInformation behaviour, which resulted in bogus file deletion handling
- fixed issue with checking WFP status
- fixed issue WFP failing to initialize at boot
- fixed issue with tray sandbox options not being available just after boot
- fixed issue access changed flag not being proeprly set in box options [#1065](https://github.com/sandboxie-plus/Sandboxie/issues/1065)

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/0.9.1'>Release v0.9.1 / 5.51.1</a>.</em>

**Comment by Coldblackice** [comment]
Awesome idea! Do you know if this would clash or conflict with another popular tool that manages WFP, "Windows Firewall Control (WFC)" (its primary discussion thread also being on Wilders)?

**Reply by DavidXanatos** [reply]
no there should be no confllicts

**Comment by Toha-Fokulov** [comment]
Hi, friend. I have been using your program for a year now. after upgrading to Windows 11, I had a problem, when I start the explorer, a window opens with a white blank, after 5 minutes everything is loaded, and it always does. I can send video or screen sharing. I think all Windows 11 users have this problem

**Reply by DavidXanatos** [reply]
yes pelase send a video
